### PR TITLE
Bug text wrapping

### PR DESCRIPTION
**Pull request by mdaxtman**
fixed bug with text wrapping in the viewer, change default font declaration.

### CODE REVIEW

File: src/components/viewer.js [hunk 1]
```diff
@@ -48,8 +64,9 @@ const renderChildren = (nodes, isListItem) =>
       );
       /* eslint-enable react/prop-types */
     }
-    if (node.props.listType) {
-      Tag = (node.props.listType === 'ordered') ? 'ol' : 'ul';
+
+    if (props.listType) {
```
Comment (rgerstenberger) [inline src/components/viewer.js:68]: why was this changed? Just code style?

**Comment by rgerstenberger** [comment]
Minor comment, LGTM!

File: src/theme/index.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ const defaultColors = {
 };
 
 const defaultFonts = {
-  primary: 'Open Sans Condensed',
+  primary: 'Open Sans',
```
Comment (mdtusz) [inline src/theme/index.js:13]: 🎉

File: src/components/viewer.js [hunk 3]
```diff
@@ -20,10 +20,26 @@ const renderChildren = (nodes, isListItem) =>
     }
     // Text node
     if (typeof node === 'string') {
-      return node;
+      const line = node.replace(/\n/g, "");
```
Comment (mdtusz) [inline src/components/viewer.js:23]: Won't this just remove line breaks completely?

Comment (mdaxtman) [inline src/components/viewer.js:23]: yes it will, but I'm nesting each line within a `span` with `display: block`, so that will force new lines without `\n`. I was getting strange side-effects if I left those line breaks in there.

File: src/components/viewer.js (hunk 1 above)
Comment (mdaxtman) [inline src/components/viewer.js:68]: yup, it was just code style. :-). Consistency is best.

File: src/theme/index.js (hunk 2 above)
Comment (mdaxtman) [inline src/theme/index.js:13]: LOL

File: src/components/viewer.js (hunk 3 above)
Comment (mdtusz) [inline src/components/viewer.js:23]: Could you just use a `<pre>`?

Comment (mdaxtman) [inline src/components/viewer.js:23]: I'll give that a shot.

Comment (mdaxtman) [inline src/components/viewer.js:23]: ah, turns out internally we're wrapping the lines in a `<p>` tag, which doesn't allow for nesting of `block` displayed elements. Theres other issues that the creation of the `<Tag>` component will account for not just text, but plotly charts and images as well. Beyond that, just wrapping the text in a `pre` tag is not preserving the line breaks for me, even when I remove any styling that come along with it. The current solution seems to be the easiest at the moment.
